bazel/foreign_cc/luajit2.patch

```diff
@@ -35,7 +35,7 @@ index 00000000..dab3606c
 +
 +    arch = subprocess.check_output(["uname","-m"]).decode("utf-8").strip()
 +    compiler = os.environ.get("CC", "")
-+    if "clang" in compiler and arch in ["s390x"]:
++    if "clang" in compiler and arch in ["ppc64le"]:
```

Comment:
should you append ppc instead of overriding s390?

Reply:
agree corrected

Reply:
@jwendell  changes done in latest commit , can you please review